whitesource.config

```diff
@@ -0,0 +1,315 @@
+###############################################################
+# WhiteSource custom configuration file
+# To enable WS integration on a FINOS repo, email help@finos.org
+# Please read docs on https://finosfoundation.atlassian.net/wiki/spaces/FDX/pages/1129283585/WhiteSource+for+GitHub.com
```

Comment:
We should link to https://odp.finos.org/docs/development-infrastructure/code-validation/whitesource instead. And also change the project blueprint accordingly of course.

Reply:
This is done and committed 👍